### PR TITLE
docs: refactor text to image example

### PR DESCRIPTION
**Pull request by jupyterjazz**
Part of https://github.com/jina-ai/finetuner/issues/451
Closes https://github.com/jina-ai/finetuner/issues/453

Refactor text to image fine-tuning example based on the new template

### CODE REVIEW

File: docs/3_finetuner_in_action/3_3_text_to_image.md [hunk 1]
```diff
@@ -1,37 +1,39 @@
 (text-to-image)=
-# Text to image search using CLIP
+# CLIP for text to image search
 
 This guide will showcase fine-tuning a `CLIP` model for text to image retrieval.
 
-## Task overview
+## Task
 We'll be fine-tuning CLIP on a fashion captioning dataset which contains information about fashion products.
```
Comment (bwanglzu) [inline docs/3_finetuner_in_action/3_3_text_to_image.md:7]: on the [fashion captioning dataset ](https://github.com/xuewyang/Fashion_Captioning)

File: docs/3_finetuner_in_action/3_3_text_to_image.md [hunk 2]
```diff
@@ -1,37 +1,39 @@
...
 
 For each product the dataset contains a title and images of multiple variants of the product. We constructed a parent `Document` for each picture, which contains two chunks: an image document and a text document holding the description of the product.
```
Comment (bwanglzu) [inline docs/3_finetuner_in_action/3_3_text_to_image.md:9]: can we link chunks to the nester document structure: https://docarray.jina.ai/fundamentals/document/nested/#nested-structure

File: docs/3_finetuner_in_action/3_3_text_to_image.md [hunk 3]
```diff
@@ -1,37 +1,39 @@
...
 
-## Preparing data
-Training and evaluation data are already prepared and pushed to Hubble following the {ref}`instructions <create-training-data>`.
-You can either pull the data:
-```python
-from docarray import DocumentArray
-train_data = DocumentArray.pull('clip-fashion-train-data')
-eval_data = DocumentArray.pull('clip-fashion-eval-data')
+## Data
+Out journey starts locally. We have to {ref}`prepare the data and push it to the cloud <create-training-data>` and Finetuner will be able to get the data by its name. For this example,
```
Comment (violenil) [inline docs/3_finetuner_in_action/3_3_text_to_image.md:13]: Our

File: docs/3_finetuner_in_action/3_3_text_to_image.md [hunk 4]
```diff
@@ -49,14 +51,20 @@ run = finetuner.fit(
...
-such as number of `epochs` and a `learning_rate`. Additionally, we use `BestModelCheckpoint` to save the best model after each epoch and `EvaluationCallback` for evaluation. Now let's move on to CLIP-specific arguments. We provided `image_modality`
-and `text_modality` which are needed for `CLIP` model to distribute data across its two models properly. (More on this in the [create training data](../2_step_by_step/2_4_create_training_data.md) section).
+Let's understand what this piece of code does:
+```{admonition} finetuner.fit parameters
+:class: tip
+The only required arguments are `model` and `train_data`. We provide default values for others. Here is the [full list of the parameters](../../api/finetuner/#finetuner.fit). 
+```
+* We start with providing `model`, `run_name`, names of training and evaluation data.
+* We also provide some hyper-parameters such as number of `epochs` and a `learning_rate`.
+* Additionally, we use `BestModelCheckpoint` to save the best model after each epoch and `EvaluationCallback` for evaluation.
```
Comment (violenil) [inline docs/3_finetuner_in_action/3_3_text_to_image.md:61]: add some developer references here for the backticks (same as other guide comments)

**Review by violenil — COMMENTED** [review]
Great! See other guide PR for more comments that are also relevant here.

**Review by bwanglzu — APPROVED** [review]
LGTM!

File: docs/3_finetuner_in_action/3_3_text_to_image.md [hunk 5]
```diff
@@ -1,37 +1,39 @@
...
 
 This guide will showcase fine-tuning a `CLIP` model for text to image retrieval.
 
-## Task overview
-We'll be fine-tuning CLIP on a fashion captioning dataset which contains information about fashion products.
+## Task
+We'll be fine-tuning CLIP on the [fashion captioning dataset](https://github.com/xuewyang/Fashion_Captioning) which contains information about fashion products.
 
-For each product the dataset contains a title and images of multiple variants of the product. We constructed a parent `Document` for each picture, which contains two chunks: an image document and a text document holding the description of the product.
+For each product the dataset contains a title and images of multiple variants of the product. We constructed a parent `Document` for each picture, which contains two [chunks](https://docarray.jina.ai/fundamentals/document/nested/#nested-structure): an image document and a text document holding the description of the product.
```
Comment (violenil) [inline docs/3_finetuner_in_action/3_3_text_to_image.md:9]: Maybe `Document` can still receive a developer reference here? It is its first mention in this guide.

**Review by violenil — COMMENTED** [review]
Looks good! Just one small comment.

**Pull request merged**